### DISCUSSION TITLE
List data type in Xandra

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [vinniefranco/exandra#39] [Q&A]
according to the xandra docs, they have a list datatype but I cant understand how does that work since there is no corresponding list datatype in cassandra. Can anyone point to me any links on how can I create a table with a list datatypes (which I need to correspond to elixir list data-type, i.e, linked list)

**Comment by whatyouhide** [comment]
In general, this would be something to report/ask in [Xandra](https://github.com/lexhide/xandra). Either way, `list<...>` is a built-in type in Cassandra: https://docs.datastax.com/en/cql-oss/3.x/cql/cql_using/useList.html

🙃
